cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -63,6 +63,10 @@ export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
     ${ksh} get node node01
     ${ksh} get node node02
 
+    # print kubelet config
```

Comment:
Don't we see this from kubeadm already?

Reply:
I couldn't find the effective config being printed anywhere else, would you mind pointing me to where this is done?